corehq/apps/app_manager/models.py

```diff
@@ -5534,10 +5534,7 @@ def create_profile(self, is_odk=False, with_media=False,
         if toggles.CUSTOM_PROPERTIES.enabled(self.domain) and "custom_properties" in self__profile:
             app_profile['custom_properties'].update(self__profile['custom_properties'])
 
-        if toggles.PHONE_HEARTBEAT.enabled(self.domain):
-            apk_heartbeat_url = self.heartbeat_url
-        else:
-            apk_heartbeat_url = None
+        apk_heartbeat_url = self.heartbeat_url
```

Comment:
@sravfeyn 

I think this is the only change we need - we still want to hide the UX for "enable update prompts" behind the feature flag.

Reply:
Cool, updated